packages/backend/src/apps/toolbox/actions/if-then/index.ts

```diff
@@ -78,7 +78,13 @@ const action: IRawAction = {
       type: 'string' as const,
       label: 'FILE A BUG IF YOU SEE THIS',
 
-      hidden: true,
+      // Always hidden
+      hiddenIf: {
+        fieldKey: '',
+        fieldValue: '',
+        op: 'always_true',
+        not: false,
```

Comment:
could you elaborate on the purpose of the `not` boolean? why do we sometimes need to invert the boolean condition? could this be handled by `op` instead?

Reply:
Good question. I directly lifted this off the way we process our conditional actions (if-then, only-continue-if) etc for ease of understanding and review (faster review = faster ship)

In reality, we don't need the `not`. Realistically we only want to support these operators:

* `===`
* `!==`
* `>=`
* `<=`
* `>`
* `<`
* `always_true`
* `always_false`

TBD:
* `starts_with`
* `not_starts_with`
* `ends_with`
* etc... 

This `not `will only reduce the number of operators by 2 (removes `always_false` and `!==`), so it's not needed. Will remove it.